packages/keymaps/src/browser/keymaps-frontend-contribution.ts

```diff
@@ -49,6 +51,33 @@ export namespace KeymapsCommands {
         id: 'keymaps.clearSearch',
         iconClass: codicon('clear-all')
     };
+    export const COPY_KEYBINDING: Command = {
```

Comment:
You can set a keybinding for those commands, so they are visible in the keybindings widget 🤷

Reply:
I'm sorry if I'm again missing something, but I still don't think these commands should have a label, as this would have made them accessible to the user from the command palette. These commands would not work correctly if invoked from the command palette, as they require specific arguments, which would not have been provided in this case.

There are similar places in the existing code where such commands don't have a label like
https://github.com/eclipse-theia/theia/blob/e0c82d13a461d1732c13852c8f1e3adc5d234d2f/packages/console/src/browser/console-contribution.ts#L25-L44 or
https://github.com/eclipse-theia/theia/blob/e0c82d13a461d1732c13852c8f1e3adc5d234d2f/packages/output/src/browser/output-commands.ts#L28-L50

It looks like a pattern to me, or am I missing something?

Reply:
Your are right. Looks like I have been misguided by existing code then.

I have just checked that the new commands, even when they do have a label, don't actually show up in the command quick-pick.

Do you think that a separate category should also be used for the new commands (e.g. 'Keybindings')?

Thank you for your guidance, I really appreciate it!

Reply:
Just for the record, it looks like the corresponding commands in VS Code don't have a label. To see it, just search for `keybindings.` in the Keyboard Shortcuts editor in VS Code. I have suddenly discovered it when trying to find out what category (if any) those commands are in VS Code.

Having said that, I agree with your point in general, and just wanted to be sure we were still on the same page in this specific case.

Reply:
> Do you think that a separate category should also be used for the new commands (e.g. 'Keybindings')?

I think VS Code uses "Preferences" as the category for keybindings-related commands. I don't see any value in behaving differently.

> it looks like the corresponding commands in VS Code don't have a label

Having a proper label for commands is the right thing to do, IMO.

Reply:
OK. I have submitted the requested changes for your consideration.